application/classes/Ushahidi/Console/PostExporter.php

```diff
@@ -83,42 +82,32 @@ protected function configure()
 		$this
 			->setName('exporter')
 			->setDescription('Export Posts')
-			->addArgument('action', InputArgument::REQUIRED, 'list, export')
-			->addOption('limit', ['l'], InputOption::VALUE_OPTIONAL, 'limit')
-			->addOption('offset', ['o'], InputOption::VALUE_OPTIONAL, 'offset')
-			->addOption('job', ['j'], InputOption::VALUE_OPTIONAL, 'job')
-			->addOption('include_header', ['ih'], InputOption::VALUE_OPTIONAL, 'include_header')
-			;
-	}
-
-	protected function executeList(InputInterface $input, OutputInterface $output)
-	{
-		return [
-			[
-				'Available actions' => 'export'
-			]
-		];
+			->addOption('limit', ['l'], InputOption::VALUE_OPTIONAL, 'limit', 100)
+			->addOption('offset', ['o'], InputOption::VALUE_OPTIONAL, 'offset', 0)
+			->addOption('job', ['j'], InputOption::VALUE_OPTIONAL, 'job', null)
+			->addOption('include_header', ['ih'], InputOption::VALUE_OPTIONAL, 'include_header', 1)
+		;
 	}
 
-	protected function executeExport(InputInterface $input, OutputInterface $output)
+	protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Just check what the parent execute method does... you might need to do some output formatting yourself now you've dropped the actions.

Reply:
./debug ./bin/ushahidi exporter --limit 200 --include_header true --job 1
[{"file":"csv\/tmp\/5acf8b46ba35e-2018-04-12.csv"}]
Apparently it's looking good 👍 and the handlereponse call is aligned with parent options